### ISSUE TITLE
ustreamer not running on startup

### ISSUE THREAD

**Issue by JimmyRigIt** [paukstelis/octoprint_deploy#65]
Ustreamer installs correctly as i can get it running manually however i must do this everytime the computer reboots. Would be amazing if it started on a service that was installed via the script!

**Comment by paukstelis** [comment]
Did you go through the 'Add Camera' step? That makes a service file that automatically starts on boot.

**Comment by JimmyRigIt** [comment]
Yep, when creating the printer. Plugged in the printer first to get the usb port, then clicked yes for the camera and plugged that in. After restarting the PC The camera doesnt boot until manually booting ustreamer via ssh. 

Running the latest stable release of debian 11 on a bmax mini pc with a n3350 processor

Thanks for the response!

**Comment by paukstelis** [comment]
What command are you giving to start ustreamer?

What is the output of `ls -la /etc/systemd/system/cam*`?

**Comment by JimmyRigIt** [comment]
**To start ustreamer cd to folder then type,** 

user@computer:~/ustreamer$ ls
docs  janus  LICENSE  linters  Makefile  man  pkg  python  README.md  src  tools  ustreamer  ustreamer-dump
user@computer:~/ustreamer$ ./ustreamer --port 8001 --host 192.168.1.1



**The output is no such file/folder exists. Cd to /etc/systemd/system and this was the list**

user@computer:/etc/systemd/system$ ls

home.service                        default.target.wants     multi-user.target.wants      sshd.service
dbus-fi.w1.wpa_supplicant1.service          display-manager.service  network-online.target.wants  suspend.target
dbus-org.freedesktop.Avahi.service          getty.target.wants       octoprint_default.service    sysinit.target.wants
dbus-org.freedesktop.ModemManager1.service  graphical.target.wants   printer.target.wants         syslog.service
dbus-org.freedesktop.nm-dispatcher.service  hibernate.target         sleep.target                 timers.target.wants
dbus-org.freedesktop.timesync1.service      hybrid-sleep.target      sockets.target.wants         ustreamer@.service

**Comment by paukstelis** [comment]
That output looks odd. Can you join the Discord server linked in the README file? It is easier to provide real-time support there.
